.github/ci_configs/CONFIG_TEMPLATE.yaml

```diff
@@ -0,0 +1,5 @@
+# A template for a CI config file. This file is used as an example for how to define a CI config file.
```

Comment:
I wonder if we need this file here, this could go into the README file of `torch_cli`